tests/integration/021_teams/main.tf

```diff
@@ -0,0 +1,22 @@
+resource "env0_team" "team_resource1" {
+  name = "team1"
+}
+
+resource "env0_team" "team_resource2" {
+  name = "team2"
+}
+
+data "env0_teams" "all_teams" {}
+
+data "env0_team" "teams" {
```

Comment:
See this integration test to understand how to use it. Let me know if something here doesn't seem right.

Reply:
LGTM